### ISSUE TITLE
sh: Can't open /home/chronos/user/Downloads/crouton

### ISSUE THREAD

**Issue by ExpertSummer328** [dnschneid/crouton#3216]
```
Please paste the output of the following command here: sudo edit-chroot -all
sudo: edit-chroot: command not found
```

#### Please describe your issue:
Trying to install unity crouton onto my Chromebook because I was having trouble with xfce and wanted to try something else.
I install crouton, go into shell, then type in "sudo sh -e ~/Downloads/crouton -t touch,unity" and it comes up with "sh: Can't open /home/chronos/user/Downloads/crouton"
I've redone that step several times now, deleting and reinstalling crouton, going into shell, typing in the same command, and getting the same error in the output.

**Comment by DennisLfromGA** [comment]
See this similar post: https://github.com/dnschneid/crouton/issues/3215#issuecomment-298200699, maybe it'll help.

**Comment by ExpertSummer328** [comment]
I've re-downloaded several times, and it hasn't worked.

**Comment by DennisLfromGA** [comment]
Please show us the output of: `ls -l ~/Downloads`

**Comment by ExpertSummer328** [comment]
`chronos@localhost / $ ls -l ~/Downloads`
`total 0`

Is there any chance I might have to format this after download?  I feel like a friend of mine did something with crouton after download when he was initially showing me this, but I'm not sure.

**Comment by ExpertSummer328** [comment]
Thanks for your help by the way.

**Comment by DennisLfromGA** [comment]
@ExpertSummer328,

> I've re-downloaded several times, and it hasn't worked.

You may have changed the *default* download location in Chrome OS settings so it's getting downloaded to another location. Here's what mine is set to:

![screenshot 2017-05-01 at 7 50 57 pm](https://cloud.githubusercontent.com/assets/3418772/25599107/8db055bc-2ea7-11e7-9b62-f7f2dea7fdef.png)

> chronos@localhost / $ ls -l ~/Downloads
total 0

Your Downloads folder is empty, no `crouton` or anything else for that matter.

If you know where it is downloaded then use that in the path instead of `~/Downloads`.

*Hope this helps,*
-**DennisL**

**Comment by ExpertSummer328** [comment]
Awesome, thank you!  I checked ask where and made sure it was in downloads and it worked.  Now:

`unity is unsupported on ARM for releases after precise.`

This came up just now.  Does this mean I have to specify a version of unity Linux, or is it just not compatible with my hardware?

**Comment by ExpertSummer328** [comment]
@DennisLfromGA I'm fairly sure I figured out was what was happening.  If you have a dual login, the shell accesses the files from the account you logged into first, regardless of which account the shell is open in.  Therefore if I downloaded crouton in my second account, then tried to access it in the shell in my second account, it wouldn't find crouton and wouldn't work.  Thanks for your help!
